packages/client/hmi-client/src/components/workflow/ops/model-config/tera-model-config-drilldown.vue

```diff
@@ -517,6 +517,8 @@ const createConfiguration = async () => {
 		return;
 	}
 
+	knobs.value.transientModelConfig = cloneDeep(data);
+	state.transientModelConfig = knobs.value.transientModelConfig;
```

Comment:
You can make it weird with just a one liner
```suggestion
	state.transientModelConfig = knobs.value.transientModelConfig = cloneDeep(data);
```